data/json/mapgen/station_radio.json

```diff
@@ -101,81 +107,78 @@
       "fill_ter": "t_flat_roof",
       "rows": [
         "                        ",
+        "          |222222222223 ",
+        "          |...........3 ",
+        "          |...........3 ",
+        "  52222222|...........3 ",
+        "  |...................3 ",
+        "  |...................3 ",
+        "  |..........A.&......3 ",
+        "  |...................3 ",
+        "  |...N.:.............3 ",
+        "  |...................3 ",
+        "  |...................3 ",
+        "  |---###----|........3 ",
+        "      #>#    |.......=3 ",
+        "             |........3 ",
+        "             |.sHЯ.3--3 ",
+        "             |.....3    ",
+        "             |-----5    ",
         "                        ",
-        "                        ",
-        "   ******               ",
-        "   ******               ",
-        "   ******               ",
-        "   ******               ",
-        "                        ",
-        " |222222223             ",
-        " |......:.3             ",
-        " |...&....3    #>#      ",
-        " |........3222#####223  ",
-        " |...................3  ",
-        " |...................3  ",
-        " |...................3  ",
-        " |...................5  ",
-        " |...................3  ",
-        " |...................3  ",
-        " |...................3  ",
-        " |...........3-------3  ",
-        " |..N...X....3          ",
-        " |...........3          ",
-        " |-----------3          ",
+        "             *******    ",
+        "             ****X**    ",
+        "             **s****    ",
+        "             *******    ",
         "                        "
       ],
       "palettes": [ "roof_palette" ],
-      "terrain": { "*": "t_tar_flat_roof", ">": "t_ladder_down" },
-      "furniture": { "N": "f_TV_antenna" },
-      "place_nested": [
-        {
-          "chunks": [
-            [ "null", 20 ],
-            [ "roof_2x2_utilities_b", 15 ],
```

Comment:
Where did these nests go?

Reply:
I removed them and just hardcoded the various rooftop fixtures into appropriate places.
Antennas over equipment rooms, AC over workspaces, water and vent over kitchen & bathroom, etc.
It adds verisimilitude and reduces the computation (very) slightly.
And the randomization added nothing, really. It's a roof. ¯\_(ツ)_/¯
Because a 2x2 square of random roof equipment isn't as good as a human placing things where humans would place them IRL.
Plus, the coordinates would all need to be transformed and bugchecked. And any future transformations or revisions would have had to as well, because 80% of the roof was defined as potential spawn points.